engines/suppliers/app/models/suppliers/suppliers_finder.rb

```diff
@@ -0,0 +1,16 @@
+module Suppliers
+  class SuppliersFinder
+
+    def initialize(group_id)
+      @group_id = group_id
+    end
+
+    def find
```

Comment:
Is this the acceptance criteria? How are we going to handle the process of finding a supplier for a group, productwise?